fastlane_core/lib/fastlane_core/ui/fastlane_runner.rb

```diff
@@ -72,5 +76,9 @@ def display_user_error!(e, message)
     def reraise_formatted!(e, message)
       raise e, "[!] #{message}".red, e.backtrace
     end
+
+    def collector
+      @collector ||= FastlaneCore::ToolCollector.new
```

Comment:
Mm, right. We discussed this as a group as well.

We don't specifically need this method/instance variable pair. We brought it over because it was done this way in the original location where the collector was used, and we wanted to keep things as consistent as possible.